docs/Guides-Renderer.md

```diff
@@ -0,0 +1,168 @@
+---
+id: guides-renderer
+layout: docs
+category: Guides
+permalink: docs/guides-renderer.html
+next: guides-ready-state
+---
+
+So far, we've covered two pieces that each contribute to declaring data:
+
+ - **Relay.queryConfig** lets us declare query roots.
```

Comment:
This looks great! Only one thing left… we don't actually export `Relay.QueryConfig`.

We should, though, and we should deprecate `Relay.Route`. See https://github.com/facebook/relay/issues/822 and https://github.com/facebook/relay/issues/503.

The way I see it, there are three things to do here:

1. Expose `Relay.QueryConfig` through `Relay.js`
2. Write a github.com/relayjs/relay-codemod that people can use to convert their routes to query configs. Add a deprecation warning to `Relay.Route` that points people there and encourages them to migrate their routes over.
3. Replace the `Relay.Route` guide with one for `Relay.QueryConfig`

What do you say. Are you bored of this yet, or would you like to keep going? If you'd like to leave it at this, you can change this line back to `Relay.Route`, mention that routes implement the `RelayQueryConfig` interface, but that you can just as soon use a plain JS object having that shape, and we'll land this PR (we should probably do that anyway). If you'd like to keep going, I can file tasks for the above and you can chip away at them.

Thanks for all of your help!